lib/adoptoposs_web/templates/layout/root.html.eex

```diff
@@ -30,5 +30,12 @@
     <footer class="bg-gray-900 text-red-400 py-6 md:py-10">
       <%= render AdoptopossWeb.SharedView, "footer.html", assigns %>
     </footer>
+
+    <div class="w-full sticky bottom-0 bg-gray-900 flex items-center justify-center py-4 cookies-box hidden" id="cookies-banner">
```

Comment:
Instead of making it sticky, I think we would probably rather use a `fixed` position. 

I guess we don't need the sticky behaviour, which would allow to keep the cookie banner in the view until it's parent container entirely scrolled out of the viewport. So, because the banner is not supposed to be scrolled up at all, a fixed position should be enough, here.